src/ontology/omo-edit.owl

```diff
@@ -185,12 +185,15 @@ We also have the outstanding issue of how to aim different definitions to differ
     <!-- http://purl.obolibrary.org/obo/IAO_0000118 -->
 
     <owl:AnnotationProperty rdf:about="http://purl.obolibrary.org/obo/IAO_0000118">
-        <obo:IAO_0000111 xml:lang="en">alternative term</obo:IAO_0000111>
+        <obo:IAO_0000111 xml:lang="en">alternative label</obo:IAO_0000111>
         <obo:IAO_0000114 rdf:resource="http://purl.obolibrary.org/obo/IAO_0000125"/>
-        <obo:IAO_0000115 xml:lang="en">An alternative name for a class or property which means the same thing as the preferred name (semantically equivalent)</obo:IAO_0000115>
+        <obo:IAO_0000115 xml:lang="en">An alternative name for a class or property which can mean the same thing as the preferred name (semantically equivalent, narrow, broad or related).</obo:IAO_0000115>
```

Comment:
Cant find physical evidence we agreed on this (I do, but that does not mean much).  Contradicts: https://github.com/information-artifact-ontology/ontology-metadata/issues/17